### ISSUE TITLE
add MySQLi driver

### ISSUE THREAD

**Issue by rakucmr** [kohana/database#91]
Why don't add MySQLi driver to the master branch, you already have it on 3.4 develop branch https://github.com/kohana/database/tree/3.4/develop/classes/Database

**Comment by shadowhand** [comment]
Because there is no 3.4 release yet.

**Comment by rakucmr** [comment]
I know that :) but why don't add the missing driver to the 3.3.3.1 release?

**Comment by enov** [comment]
@raku this might go into a 3.3.4 release.

@acoulton [do you still think](https://github.com/kohana/core/issues/547#issuecomment-64544378) we should make a 3.3.4 release with the MySQLi driver for the database module as well as Redis and XCache drivers for the cache modules?

**Comment by rakucmr** [comment]
I already use it on 3.3.3.1, I've just copied the missing driver files.

**Comment by acoulton** [comment]
@enov yes, I think a 3.3.4 is a good idea, and would be good to add this.

**Comment by ghost** [comment]
I solved this trouble by next way (i am not on 3.4 ofc):

![4](https://cloud.githubusercontent.com/assets/1884457/5958251/9af8f952-a7d3-11e4-9727-f881d4e60a81.jpg)

Where is "modules/coll3ctor" - my own module with my helpers, views, etc. So, MySQLi files in my module, and Kohana understand that and correctly use mysqli extension. And, of course, database module not touched. 

And, ofc, by this way you can use MySQLi driver in your config/database.php

**Comment by enov** [comment]
@coll3ctor :+1:

**Comment by nik045** [comment]
:+1:
